composer.json

```diff
@@ -24,7 +24,8 @@
         "illuminate/database": "^10.0|^11.0",
         "hashids/hashids": "^4.0|^5.0",
         "whichbrowser/parser": "^2.1",
-        "ashallendesign/laravel-config-validator": "^2.6.1"
+        "ashallendesign/laravel-config-validator": "^2.6.1",
+        "laravel/framework": "^10.0"
```

Comment:
Do we really need this depedency? Wouldn't this mean that it would conflict with Laravel 11 installs?

Reply:
I basically can't upgrade to `^8.2` because of this.

Reply:
Hey @kyryl-bogach! Huge thank you for pointing this out to me. I was doing some tinkering the other day and accidentally committed this (I can't believe I did that!) 🤦‍♂️

I've just removed it now, so you should be able to upgrade to v8.2.1 🤞

Sorry about that!

Reply:
Thank you so much sir! 🙏🏻